### DISCUSSION TITLE
Volume control not working on ASUS ROG Swift PG32UQX

### DISCUSSION THREAD

**Discussion by ghost** [MonitorControl/MonitorControl#1268] [General]
### Before opening the issue, have you...?

- [X] Searched for existing issues
- [X] Read through [the Monitor-Troubleshooting Wiki](https://github.com/MonitorControl/MonitorControl/wiki/Monitor-Troubleshooting)
- [X] Updated MonitorControl to the latest version (if applicable)

### Describe the issue

Volume control not working on ASUS ROG Swift PG32UQX. Brightness is working, but it alters the max brightness incorrectly.

### Expected behavior

Volume used to work on my LG Monitor.

### Environment Information (please complete the following information)

```markdown
- macOS version: macOS Ventura 13.1
- Mac model: MacBook Air M2
- MonitorControl version: 4.1.0
- Monitor(s): 1
- Monitor Cable(s)/Connection(s): Displayport to Thunderbolt 4 dock, Thunderbolt 4 to Mac
- Apple Silicon/M1 (yes or no): yes
```

**Comment by hongyoudan** [comment]
My Benq monitor also has built-in speakers that can't be adjusted.

**Comment by waydabber** [comment]
A lot of displays do not support volume controls via DDC. If brightness control works only until 50% (if this is what you mean by the max brightness issue) then it is a good indication that DDC control generally does not work (it might be your dock) as the 0-50% range of the brightness slider is by default software dimming, not hardware backlight control.
